tests/sqllogictests/suites/mode/standalone/explain/merge_limit.test

```diff
@@ -0,0 +1,156 @@
+# Test cases for RuleMergeLimit optimization rule
+# This rule merges consecutive LIMIT operators into a single LIMIT
+
+statement ok
+CREATE TABLE IF NOT EXISTS test_table (id INT, name STRING, value INT);
+
+
+query T
+EXPLAIN SELECT * FROM (SELECT * FROM test_table LIMIT 10) LIMIT 5;
+----
+Limit
+├── output columns: [test_table.id (#0), test_table.name (#1), test_table.value (#2)]
+├── limit: 5
+├── offset: 0
+├── estimated rows: 0.00
+└── TableScan
+    ├── table: default.default.test_table
+    ├── output columns: [id (#0), name (#1), value (#2)]
+    ├── read rows: 0
+    ├── read size: 0
+    ├── partitions total: 0
+    ├── partitions scanned: 0
+    ├── push downs: [filters: [], limit: 10]
```

Comment:
I'm currently having a problem. Because the rewrite rules are bottom-up, for example, if I encounter `Limit(5)-Limit(10)-Scan`, it will first become `Limit(5)-Limit(10)-Scan(10)` and finally become `Limit(5)-Scan(10)`. This doesn't affect the correctness of the result, but it does make the rewrite less thorough. Could you offer some advice? Or is this method acceptable for now?